Innovative Flix/force-app/main/default/objects/Episode__c/listViews/All.listView-meta.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Add some fields to list view ;)

Reply:
Done

Reply:
There are no episodes in database, so I can't check how the list view looks like

Reply:
Episodes added